test/instance.ts

```diff
@@ -258,7 +269,7 @@ describe('Instance', () => {
     it('should accept gaxOptions', done => {
       const options = Object.assign({}, OPTIONS, {gaxOptions: {}});
       instance.request = config => {
-        assert.strictEqual(config.gaxOpts, options.gaxOptions);
+        assert.deepStrictEqual(config.gaxOpts, options.gaxOptions);
```

Comment:
```suggestion
        assert.strictEqual(config.gaxOpts, options.gaxOptions);
```

If after making that change, the test fails, then we would want to keep `deepStrictEqual`, but also set a property on `options.gaxOptions`.